pod-master/podmaster.go

```diff
@@ -138,6 +141,9 @@ func copyFile(src, dest string) error {
 
 func initFlags(c *config) {
 	pflag.StringVar(&c.etcdServers, "etcd-servers", "", "The comma-seprated list of etcd servers to use")
+	pflag.StringVar(&c.etcdCA, "etcd-ca", "", "The CA file for etcd tls.")
```

Comment:
add these to https://github.com/kubernetes/contrib/blob/master/hack/verify-flags/known-flags.txt